test/msw-api/setup-server/scenarios/concurrency/shared-server.ts

```diff
@@ -0,0 +1,43 @@
+import { setupServer } from 'msw/node'
+
+export const sharedServer = setupServer()
+
+export const sleepDelay = 3000
+export const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
```

Comment:
We already have the `sleep` utility, you can reuse it:

https://github.com/mswjs/msw/blob/b93aca9f165294d064517990e6c55aee7557d9b1/test/support/utils.ts#L4